pypeit/scripts/collate_1d.py

```diff
@@ -425,8 +431,8 @@ def coadd(par, coaddfile, source):
     # Set destination file for coadding
     par['coadd1d']['coaddfile'] = coaddfile
     
-    # Whether to be forgiving of data model versions
-    par['coadd1d']['chk_version'] = par['collate1d']['chk_version']
+#    # Whether to be forgiving of data model versions
+#    par['coadd1d']['chk_version'] = par['collate1d']['chk_version']
```

Comment:
This is being lost in the call to `coadd1d.CoAdd1D.get_instance` on line 458 below (although, I'm not really sure it's even needed...)

also, remove these commented lines?